src/modules/esl-random-text/test/esl-random-text.test.ts

```diff
@@ -0,0 +1,114 @@
+import {ESLRandomText} from '../core/esl-random-text';
+
+describe('ESLRandomText', () => {
+  const normalizeWord = (word: string) => word.toLowerCase().replace(/[.,]/g, '');
+
+  describe('ESLRandomText.generateText', () => {
+    test('generates text with specified number of words', () => {
+      const text = ESLRandomText.generateText(10);
+      expect(text.split(' ').length).toBe(10);
+    });
+
+    test('generates text with specified number of words and shuffles them', () => {
```

Comment:
It checks if the shuffle does not break the count of words in the output
As you can see below the shuffling itself is not so simple to check because we need to reduce the probability of the same output (as it is exists)